mask/requirements.txt

```diff
@@ -0,0 +1,143 @@
+aenum==3.1.11
```

Comment:
Is this your full environment, or is this just what is needed?

I'd assume fixing `eo-learn` and `sentinelhub-py`, and adding `jupyter` might be enough?

Reply:
This was a discussion we had with Jan a couple of times but found no real 'winner'.
1. Freezing the minimal environment. Not readable or flexible, but should in theory work for a long long time.
2. Fixing our packages. Much more readable and can be used with existing environments. Breaks if any of the dependencies has a codebreaking update.

Reply:
Indeed. I'm ok with 1, but to some extent would prefer 2. If you'd already had a vote on it, just resolve this comment.

Reply:
" fixing eo-learn and sentinelhub-py, and adding jupyter might be enough?" 

it is

Reply:
fix s2cloudless as well please

Reply:
ah, I see the models are used directly in the eolearn.mask module, so no s2cloudless dependency.
smells a bit, will investigate